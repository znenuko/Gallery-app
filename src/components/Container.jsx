import React from "react";
import Cards from "./Cards";

const Container = () => {
  return (
    <div aria-label="container" className="container border my-3">
      <Cards />
    </div>
  );
};

export default Container;
