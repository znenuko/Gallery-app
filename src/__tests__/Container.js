import { render, screen } from "@testing-library/react";
import Container from "../components/Container";

describe("<Container />", () => {
  it("Snapshot principal", () => {
    render(<Container />);

    expect(screen.getByLabelText("container")).toMatchSnapshot();
  });
});
