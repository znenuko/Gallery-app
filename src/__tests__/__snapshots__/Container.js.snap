// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> Snapshot principal 1`] = `
<div
  aria-label="container"
  class="container border my-3"
>
  <div
    class="text-center"
  >
    <form>
      <label
        class="w-75"
      >
         
        Buscar: 
        <input
          class="w-100"
          name="inputText"
          type="text"
        />
         
      </label>
      <button
        class="btn btn-warning m-2"
        type="submit"
      >
        <span
          class="material-icons"
        >
          search
        </span>
      </button>
    </form>
    <hr />
    <div
      class="spinner-border text-info"
      role="status"
    >
      <span
        class="visually-hidden"
      >
        Loading...
      </span>
    </div>
    <div
      aria-label="imagenes"
      class="row"
    />
  </div>
</div>
`;
